spec/domain/people/neuanmeldungen/promoter_spec.rb

```diff
@@ -284,4 +290,19 @@ def create_neuanmeldung_role(family_main_person: true, **opts)
       subject.call
     end
   end
+
+  context "role definitions" do
+    it "excludes tourenportal roles from OBSOLETE_ROLES" do
+      expect(described_class::OBSOLETE_ROLES).not_to include(Group::AboTourenPortal::Abonnent.sti_name)
+      expect(described_class::OBSOLETE_ROLES).not_to include(Group::AboTourenPortal::Neuanmeldung.sti_name)
+    end
+
+    it "includes only magazine registration in NEUANMELDUNG_ROLES" do
+      expect(described_class::NEUANMELDUNG_ROLES).to contain_exactly(Group::AboMagazin::Neuanmeldung.sti_name)
+    end
+
+    it "excludes membership registrations from NEUANMELDUNG_ROLES" do
+      expect(described_class::NEUANMELDUNG_ROLES).not_to include(*SacCas::NEUANMELDUNG_ROLES.map(&:sti_name))
+    end
+  end
```

Comment:
I'm not sure how much value these specs provide, since they just mirror the code almost exactly.